packages/components/data-table/src/data-table.spec.js

```diff
@@ -12,10 +12,12 @@ const testColumns = [
   {
     key: 'title',
     label: 'Title',
+    isTruncated: true,
   },
   {
     key: 'year',
     label: 'Year',
+    isTruncated: true,
```

Comment:
Do you think we should keep this change the story, or it was just for testing? In practice, it almost makes no difference anyway :)

Reply:
These changes were on purpose to test the expand/collapse button rendering. We could create another set of test data, but to me that looked a bit redundant. Therefore I updated the `default` test data to check the rendering of the expand/collapse buttons.

Reply:
Oh, nevermind, I thought I was reading the `.story.js` file 🤦‍♂️ 
All good!